docs/servicenow.md

```diff
@@ -0,0 +1,143 @@
+# ticketutil.servicenow
+
+This document contains information on the methods available when working
+ with a ServiceNow object. A list of the ServiceNow fields that have been
+ tested when creating and editing tickets will be included. Because each
```

Comment:
> A list of the ServiceNow fields that have been tested when creating and editing tickets will be included.

Is this some kind of future feature promise? I think the documentation should contain only description of the current features.